app/controllers/concerns/hyku_addons/catalog_controller_behavior.rb

```diff
@@ -15,6 +15,8 @@ module CatalogControllerBehavior
         config.add_facet_field solr_name('member_of_collections', :symbol), limit: 5, label: 'Collections'
         config.add_facet_field solr_name("institution", :facetable), limit: 5, label: 'Institution'
         config.add_facet_field solr_name("language", :facetable), limit: 5, label: 'Language'
+        config.add_facet_field solr_name("org_unit", :facetable), limit: 5, label: 'Department'
+        config.add_facet_field solr_name("audience", :facetable), limit: 5, label: 'OER Audience', if: proc { |context, _config, _opts| context.current_account.locale == "redlands" }
```

Comment:
It looks like this setting is actually `locale_name` so we could update this based upon other examples in the code:
```suggestion
        config.add_facet_field solr_name("audience", :facetable), limit: 5, label: 'OER Audience', if: proc { |context, _config, _opts| context.current_account&.settings&.dig("locale_name") == "redlands" }
```
Or using settings:
```suggestion
        config.add_facet_field solr_name("audience", :facetable), limit: 5, label: 'OER Audience', if: proc { |_context, _config, _opts| Settings.locale_name == "redlands" }
```